runbot_custom_demo_data/runbot_build.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Author: Samuel Lefever
+#    Copyright 2015 Niboo SPRL
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import os
+import time
+import logging
+import openerp
+from openerp.fields import Boolean
+from openerp.models import api, Model
+
+_logger = logging.getLogger(__name__)
+
+
+def grep(filename, string):
+    if os.path.isfile(filename):
+        return open(filename).read().find(string) != -1
+    return False
+
+
+def now():
+    return time.strftime(openerp.tools.DEFAULT_SERVER_DATETIME_FORMAT)
+
+
+class RunbotRepo(Model):
+
+
+    _inherit = 'runbot.repo'
+
+    custom_demo_data = Boolean(
+        'Load custom data', default=False,
+        help="Load only the demo data from this repository")
+
+
+class RunbotBuild(Model):
+
+
+    _inherit = "runbot.build"
+
+    @api.model
+    def job_15_install_all(self, build, lock_path, log_path):
+        """
+        Install needed modules and dependencies without any demo data
+        """
+        if build.repo_id.custom_demo_data:
+            build._log('install_all', 'Start install all modules')
+            self.pg_createdb("%s-all" % build.dest)
```

Comment:
Why you are duplicating database name of [job_20](https://github.com/odoo/odoo-extra/blob/80722c03def52c8c1876d78ef3e805531562e8cb/runbot/runbot.py#L883)?